administrator/templates/atum/html/layouts/chromes/well.php

```diff
@@ -35,35 +35,50 @@
 	// Temporarily store header class in variable
 	$headerClass = $params->get('header_class');
 	$headerClass = ($headerClass) ? ' ' . htmlspecialchars($headerClass) : '';
+
+	// Get the module icon
+	$headerIcon = '';
+	
+	$margin = Factory::getLanguage()->isRtl() ? ' ml-2' : ' mr-2';
+	
+	if (!empty($params->get('header_icon')))
+	{
+		$headerIcon = '<span class="' . htmlspecialchars($params->get('header_icon')) .  $margin . '" aria-hidden="true"></span>';
+	}
 	?>
 	<div class="module-wrapper">
 		<<?php echo $moduleTag; ?> class="card mb-3<?php echo $moduleClassSfx; ?>">
-			<?php if ($canEdit || $canChange) : ?>
-				<?php $dropdownPosition = Factory::getLanguage()->isRTL() ? 'left' : 'right'; ?>
-				<div class="module-actions dropdown">
-					<button type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" class="btn btn-link" id="dropdownMenuButton-<?php echo $id; ?>">
-						<span class="fa fa-cog"><span class="sr-only">
-							<?php echo Text::_('JACTION_EDIT') . ' ' . $module->title; ?>
-						</span></span>
-					</button>
-					<div class="dropdown-menu dropdown-menu-<?php echo $dropdownPosition; ?>" aria-labelledby="dropdownMenuButton-<?php echo $id; ?>">
-						<?php if ($canEdit) : ?>
-							<?php $uri = Uri::getInstance(); ?>
-							<?php $url = Route::_('index.php?option=com_modules&task=module.edit&id=' . $id . '&return=' . base64_encode($uri)); ?>
-							<a class="dropdown-item" href="<?php echo $url; ?>"><?php echo Text::_('JACTION_EDIT'); ?></a>
-						<?php endif; ?>
-						<?php if ($canChange) : ?>
-							<button type="button" class="dropdown-item unpublish-module" data-module-id="<?php echo $id; ?>"><?php echo Text::_('JACTION_UNPUBLISH'); ?></button>
-						<?php endif; ?>
-					</div>
-				</div>
-			<?php endif; ?>
+			<?php if ($canEdit || $canChange || $headerIcon || $module->showtitle) : ?>
+				<div class="card-header">
+					<?php if ($canEdit || $canChange) : ?>
+						<?php $dropdownPosition = Factory::getLanguage()->isRTL() ? 'left' : 'right'; ?>
+						<div class="module-actions dropdown">
+							<button type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" class="btn btn-link" id="dropdownMenuButton-<?php echo $id; ?>">
+								<span class="fa fa-cog"><span class="sr-only">
```

Comment:
Please change so that the icon is hidden from screen readers as we have done everywhere else in Joomla